resources/views/cart/partials/product/description.blade.php

```diff
@@ -1,23 +1,23 @@
 <td class="flex max-md:w-1/2 md:table-cell">
     @include('rapidez-ct::cart.partials.product.alert')
     <div class="flex flex-col items-start">
-        <a :href="item.product.url_key + item.product.url_suffix | url">
-            <div dusk="cart-item-name">@{{ item.product.name }}</div>
+        <div>
+            <a :href="item.product.url_key + item.product.url_suffix | url" dusk="cart-item-name">@{{ item.product.name }}</a>
             <div v-for="option in item.configurable_options">
                 @{{ option.option_label }}: @{{ option.value_label }}
             </div>
             <div v-for="option in item.customizable_options">
                 @{{ option.label }}: @{{ option.values[0].label || option.values[0].value }}
             </div>
-            <div class="text-red-600" v-if="!canOrderCartItem(item)">
-                @lang('This product it out of stock, remove it to continue your order.')
+            <div v-if="!canOrderCartItem(item)" class="text-ct-error text-balance bg-ct-error/5 my-2 max-w-sm p-2">
```

Comment:
Please replace the v-if with this check; https://github.com/rapidez/core/pull/627/files#diff-9da9c7eb5d8ffc35371000ac0ceb2c0f5b43725a9dfc4312be677e84a22cf3dbR18